.github/workflows/pr-snapshot.yaml

```diff
@@ -18,35 +20,34 @@ jobs:
         run: |
           pr_number="$(head -n 2 /tmp/artifacts/metadata.txt | tail -n 1)"
           pr_commit="$(tail -n 1 /tmp/artifacts/metadata.txt)"
-          echo PR_COMMIT="$pr_commit" >> "$GITHUB_OUTPUT"
-          echo PR_NUMBER="$pr_number" >> "$GITHUB_OUTPUT"
+          echo PR_COMMIT="$pr_commit" >> "GITHUB_ENV"
+          echo PR_NUMBER="$pr_number" >> "GITHUB_ENV"
```

Comment:
This doesn't need a $? Couldn't find the docs for this, so hoping you can point me.

Reply:
You're completely correct. That's a miss on my side. Good catch.